gulp/metalsmith.js

```diff
@@ -57,6 +57,12 @@ function metalsmith() {
     }
     return null;
   });
+  handlebars.registerHelper("equals", function(a, b, options) {
```

Comment:
If we're manually creating handlebars helpers like this we should simply include a pre-made library. I strongly recommend avoiding custom Handlebars helpers as they tend to result in a lot of code bloat.

https://github.com/helpers/handlebars-helpers